utils/build/build-playwright-driver.sh

```diff
@@ -47,8 +47,28 @@ function build {
   cp ./output/${NODE_DIR}/LICENSE ./output/playwright-${SUFFIX}/
   cd ./output/playwright-${SUFFIX}/package
   PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD=1 node "../../${NODE_DIR}/${NPM_PATH}" install --production
+
+  # ----------- Minifying magic ----------
+  # We mess up with lib files location, and therefore use _PW_PACKAGE_ROOT
+  # environment variable to guide our code (see packageRoot() in src).
+
+  # The goal here is to have "lib2/cli/cli.js" and "lib2/cli/traceViewer/web/*".
+  npm install -D @vercel/ncc@0.26.2
```

Comment:
This would change root `package.json` unless there's a `package.json` in this folder.

How about:
```
npm init -y && npm install -D @vercel/ncc@0.26.2
```